shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -823,3 +823,28 @@ async fn events_keyspace(#[case] driver: CassandraDriver) {
 
     shotover.shutdown_and_then_consume_events(&[]).await;
 }
+
+#[rstest]
+#[case::cdrs(CdrsTokio)]
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_protocol_v5(#[case] driver: CassandraDriver) {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yaml");
+
+    let shotover = ShotoverProcessBuilder::new_with_topology(
+        "example-configs/cassandra-passthrough/topology.yaml",
+    )
+    .start()
+    .await;
+
+    let connection = || {
+        CassandraConnectionBuilder::new("127.0.0.1", 9042, driver)
+            .with_protocol_version(ProtocolVersion::V5)
```

Comment:
Can we make variants of this test for v3 and v4?
Currently V4 is the default for cdrs so we do get coverage that way but there is no gaurantee that V4 will remain the default in the future.